docs/UserGuide.md

```diff
@@ -279,7 +279,7 @@ The task Add parser has its priority changed to:
         High priority
 ```
 
-#### 3.3.5. Mark Task(s) as Complete
+#### 3.3.5. <a id="mark-task-as-complete">Mark Task(s) as Complete</a>
```

Comment:
Mj, can help to correct this?
* This header does not tally with the TBC
* Not consistent with other headers like "Add Tasks to the Sprint"

Reply:
Ok I've fixed the inconsistencies.